internal/host/validator.go

```diff
@@ -1552,3 +1552,58 @@ func (v *validator) compatibleAgent(c *validationContext) (ValidationStatus, str
 	return ValidationFailure, "This host's agent is in the process of being upgraded to a compatible " +
 		"version. This might take a few minutes"
 }
+
+func (v *validator) noSkipInstallationDisk(c *validationContext) (ValidationStatus, string) {
+	const (
+		successMessage string = "No request to skip formatting of the installation disk"
+		failureMessage string = "Requesting to skip the formatting of the installation disk is not allowed. The installation disk has to be formatted. Please either change this host's installation disk or don't skip the formatting of the installation disk"
+	)
+
+	// Make sure the user didn't ask us to skip the formatting of the current
+	// installation disk
+	if c.host.InstallationDiskID == "" || !funk.ContainsString(
+		common.GetSkippedFormattingDiskIdentifiers(c.host), c.host.InstallationDiskID) {
+
+		return ValidationSuccess, successMessage
+	}
+
+	return ValidationFailure, failureMessage
+}
+
+func (v *validator) noSkipMissingDisk(c *validationContext) (ValidationStatus, string) {
```

Comment:
This should never happen because the identifiers that we now use are hardware-based, but I appreciate your paranoia :)